meson.build

```diff
@@ -821,7 +827,7 @@ elif meson.get_compiler('c').get_id() == 'clang'
   # Clang does not support spec files
 else
   test_link_depends += test_specs
-  test_c_args += ['--specs', '@0@/@1@'.format(meson.current_build_dir(), test_specs)]
```

Comment:
Ah, note that this instance uses a different specs file than the first instance. I suspect that's why we're seeing some test failures from this patch.

Reply:
Ah, yes, my limited rv32 test only uses the other instance. I've pushed a better variant.